src/System.Private.Xml/src/System/Xml/BinaryXml/SqlUtils.cs

```diff
@@ -65,11 +65,20 @@ public BinXmlSqlDecimal(byte[] data, int offset, bool trim)
             byte b = data[offset];
             switch (b)
             {
-                case 7: m_bLen = 1; break;
-                case 11: m_bLen = 2; break;
-                case 15: m_bLen = 3; break;
-                case 19: m_bLen = 4; break;
-                default: throw new XmlException(SR.XmlBinary_InvalidSqlDecimal, (string[])null);
+                case 7:
```

Comment:
I ran CodeFormatter here to make sure running CodeRemover didn't leave the code in bad style. Isn't the format given by CodeFormatter what we prefer? If everyone agrees it's not necessary, I will skip all the style changes in this PR.